x-pack/plugins/observability_solution/synthetics/common/runtime_types/certs.ts

```diff
@@ -51,6 +51,7 @@ export const CertType = t.intersection([
     '@timestamp': t.string,
     serviceName: t.string,
     errorMessage: t.string,
+    errorStackTrace: t.union([t.string, t.null]),
```

Comment:
Why do we need to add `t.null`? I am trying to understand how it is different, for example, from errorMessage.

Reply:
because, it can also be null in heartbeat data

Reply:
So, in heartbeat data, errorMessage is always present but not errorStackTrace? Where do you check the heartbeat data type?

Reply:
no both can be optional, but error_stack can also be null